base/threadcall.jl

```diff
@@ -0,0 +1,84 @@
+const max_ccall_threads = parse(Int, get(ENV, "UV_THREADPOOL_SIZE", "4"))
+const thread_notifiers = [Nullable{Condition}() for i in 1:max_ccall_threads]
```

Comment:
filling this with async work will starve the libuv queue and leave it unable to process some types of IO operations. there should be a note in the documentation to that effect